client/apps/employee/.env.test.example

```
@@ -0,0 +1,15 @@
+# E2E Test Environment Variables
+# Copy this to .env.test for local E2E testing
+
+# App URL for E2E tests
+VITE_APP_URL=http://localhost:5173
+
+# API URL for E2E tests
+VITE_API_URL=http://localhost:3000/api/v1
+
+# Test user credentials (should match seeded data)
+E2E_TEST_EMPLOYEE_USERNAME=test_employee
+E2E_TEST_EMPLOYEE_PASSWORD=TestPassword123!
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The E2E_TEST_EMPLOYEE_PASSWORD key should go before the E2E_TEST_EMPLOYEE_USERNAME key

(UnorderedKey)